packages/matter.js/src/cluster/ClusterInterface.ts

```diff
@@ -0,0 +1,13 @@
+import { Cluster, Attributes, Commands, Events } from "./Cluster.js";
+
+/**
+ * Instances of this class tie together the public interface types and the
+ * definition for a specific cluster.  They are generated in the cluster
+ * interface file and consumed in standard device implementations.
+ */
+export type ClusterInterface<State, Client, Server> = {
```

Comment:
should we use the same notation like in other places with 

ST extends State, CL extends Client, SRV extends Server

and then use these below?

Reply:
Sure, will change.  I know cryptic generic parameters are pretty standard.  As a relative newcomer to the library I feel obliged to mention that the numerous appearances of "F, SF, A, C, E" seemed opaque and took a bit to understand. 😉  I'll try to ameliorate using little comments.